observability-kit-agent/src/main/java/com/vaadin/extension/instrumentation/client/ClientInstrumentation.java

```diff
@@ -46,8 +47,12 @@ public void transform(TypeTransformer transformer) {
     }
 
     public static class MethodAdvice {
+
+        static final String FRONTEND_ID = "vaadin.front-end.id";
```

Comment:
Nit: elsewhere we spell `frontend` and `backend` without the hyphen.
```suggestion
        static final String FRONTEND_ID = "vaadin.frontend.id";
```